src/components/GuestList/GuestList.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import Guest from '../Guest/Guest';
+
+type NewGuest = {
+  id: number,
+  guestName: string,
+  phoneNumber: number
+}
+
+const GuestList: React.FC = () => {
+  
+  const [guestName, setGuestName] = useState('')
+  const [phoneNumber, setPhoneNumber] = useState(0)
+  const [guests, setGuests] = useState<NewGuest[]>([])
+
+  const submitGuest = (event: React.FormEvent) => {
+    event.preventDefault();
+    const newGuest: NewGuest = {
+      id: Date.now(),
+      guestName,
+      phoneNumber
+    }
+    setGuests([...guests, newGuest])
+    // should be a POST request + adding card to UI
+    clearInputs();
+  }
+
+  const clearInputs = () => {
+    setGuestName('')
+    setPhoneNumber(0)
+  }
+
+  const deleteGuest = (id: number) => {
```

Comment:
Can we add an option to edit the guest as well?

Reply:
That would be something cool to look into!